src/PackagePublisher/Program.cs

```diff
@@ -24,7 +24,9 @@ public static class Program
 
         // Do NOT log the key
         private static string _apiKey = null;
-        private static int _maxRetryCount = 10;
+        private static int _maxRetryCount = 5;
+        private static int _packagePushTimeoutInSeconds = 90;
```

Comment:
Earlier timeout of 30 seconds for an individual package push seems less considering we have packages of size ~90mb, but I reduced the number of retries.